manifests/r/Rowley/CrossWorks/4.8.3/Rowley.CrossWorks.installer.yaml

```
@@ -0,0 +1,39 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Rowley.CrossWorks
+PackageVersion: 4.8.3
```

```suggestion
PackageVersion: 4.8
```
This must match what is shown in Control Panel